common-crafting.lic

```diff
@@ -106,9 +106,17 @@ module DRCC
   def get_crafting_item(name, bag, bag_items, belt)
     waitrt?
     if belt && belt['items'].find { |item| /\b#{name}/i =~ item || /\b#{item}/i =~ name }
```

Comment:
I just downloaded SmartGit. Would it be easier/clearer if I delete this pull request and resubmit using SmartGit?